Cargo.toml

```diff
@@ -22,6 +23,7 @@ serde_json = "1.0"
 log = "0.4"
 env_logger = "0.8"
 tokio = { version = "1.4", features = ["macros", "rt-multi-thread"] }
+derive_more = "0.99"
```

Comment:
Anyhow’s error already has `Display`, why should `derive_more` be introduced?

Reply:
Anyhow requires error implied `std::error::Error` which needs to derive `Display` trait firstly.

Reply:
I thought anyhow exported the derive Error macro of thiserror, but not :(